logger/logger_setup.go

```diff
@@ -134,24 +134,24 @@ func registerNewSubLogger(logger string) *subLogger {
 
 // register all loggers at package init()
 func init() {
-	Global = registerNewSubLogger("log")
-
-	ConnectionMgr = registerNewSubLogger("connection")
-	CommunicationMgr = registerNewSubLogger("comms")
-	ConfigMgr = registerNewSubLogger("config")
-	DatabaseMgr = registerNewSubLogger("database")
-	OrderMgr = registerNewSubLogger("order")
-	PortfolioMgr = registerNewSubLogger("portfolio")
-	SyncMgr = registerNewSubLogger("sync")
-	TimeMgr = registerNewSubLogger("timekeeper")
-	WebsocketMgr = registerNewSubLogger("websocket")
-	EventMgr = registerNewSubLogger("event")
-	DispatchMgr = registerNewSubLogger("dispatch")
-
-	ExchangeSys = registerNewSubLogger("exchange")
-	GRPCSys = registerNewSubLogger("grpc")
-	RESTSys = registerNewSubLogger("rest")
-
-	Ticker = registerNewSubLogger("ticker")
-	OrderBook = registerNewSubLogger("orderbook")
+	Global = registerNewSubLogger("LOG")
+
+	ConnectionMgr = registerNewSubLogger("CONNECTION")
+	CommunicationMgr = registerNewSubLogger("COMMS")
+	ConfigMgr = registerNewSubLogger("CONFIG")
+	DatabaseMgr = registerNewSubLogger("DATABASE")
+	OrderMgr = registerNewSubLogger("ORDER")
+	PortfolioMgr = registerNewSubLogger("PORTFOLIO")
+	SyncMgr = registerNewSubLogger("SYNC")
+	TimeMgr = registerNewSubLogger("TIMEKEEPER")
+	WebsocketMgr = registerNewSubLogger("WEBSOCKET")
+	EventMgr = registerNewSubLogger("EVENT")
+	DispatchMgr = registerNewSubLogger("DISPATCH")
+
+	ExchangeSys = registerNewSubLogger("EXCHANGE")
+	GRPCSys = registerNewSubLogger("GRPC")
+	RESTSys = registerNewSubLogger("REST")
+
+	Ticker = registerNewSubLogger("TICKER")
+	OrderBook = registerNewSubLogger("ORDERBOOK")
```

Comment:
While I don't really mind that these are changed to upper case can we either add support for migrating existing to upper case or even something as simple as strings.ToUpper in registerNewSubLogger 

(I have a config setup already with some subloggers configured :D)